webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -178,7 +195,17 @@ export default {
   border-top: $border-size-base solid $border-color-softest;
 }
 
-.contribution-image {
-  max-height: 300px;
+.crop-overlay {
+  max-height: 2000px;
+  position: relative;
+  width: 100%;
+  background-color: #000;
+}
+
+.crop-confirm {
+  position: absolute;
+  left: 10px;
+  top: 10px;
+  z-index: 1;
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-14T17:00:50Z" title="Monday, October 14th 2019, 7:00:50 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -178,7 +195,17 @@ export default {
   border-top: $border-size-base solid $border-color-softest;
 }
 
-.contribution-image {
-  max-height: 300px;
+.crop-overlay {
+  max-height: 2000px;
+  position: relative;
+  width: 100%;
+  background-color: #000;
+}
+
+.crop-confirm {
+  position: absolute;
+  left: 10px;
+  top: 10px;
+  z-index: 9999;
```


```suggestion
  z-index: 1;
```

this puts the button on top of the overlay but under the header ;)